test/Mono.Linker.Tests.Cases/Warnings/WarningSuppression/SuppressWarningsInCompilerGeneratedCode.cs

```diff
@@ -344,55 +348,62 @@ class SuppressInLambda
 		{
 			// Suppression currently doesn't propagate to local functions
 
-			[ExpectedWarning ("IL2026", CompilerGeneratedCode = true)]
+			[ExpectedWarning ("IL2026", CompilerGeneratedCode = true, ProducedBy = ProducedBy.Trimmer)]
```

Comment:
I'm puzzled by this one, here the warning should be produced by the RequiresAnalyzerBase/RequiresUnreferencedCodeAnalyzer not having to do anything with the change in the dataflow analyzer. I'm not certain why this stopped warning. Seems more related to the discard operation change than the dataflow change.

Reply:
The analyzer doesn't produce a warning since there is an `UnconditionalSuppressMessage `attribute on that method

Reply:
Then the question reversed, dataflow will not do anything with the call AFAIK. The dataflow only produces IL2026 when using the binder and this is not the case, why it was warning before? Do we have a problem with the testing infra?